src/backend/src/services/work-packages.services.ts

```diff
@@ -729,4 +730,66 @@ export default class WorkPackagesService {
 
     return workPackageTemplateTransformer(workPackage);
   }
+
+  static async editWorkPackageTemplate(
```

Comment:
Could you make this return the edited WPT and then return it through the 200 status in controller